### ISSUE TITLE
Nicer 0 results search page

### ISSUE THREAD

**Issue by razzeee** [flathub/frontend#138]
Not sure what we want to do, but a nice graphic would be cool

**Comment by nedrichards** [comment]
Yeah, we need an empty state. I'm adding this to my list of release blockers.

**Comment by razzeee** [comment]
I think we should also add a link to https://discourse.flathub.org/t/about-the-requests-category/22 on this page, asking for users to request apps they can't find.

**Comment by nedrichards** [comment]
That's a good idea. It'll be manual but some looking at the top empty search stats could result in bug reports for apps that have poorly optimised appdata for SEO!

**Comment by nedrichards** [comment]
So this is a 'nice to have a graaphic' currently? I don't think that's a full on release blocker since this is already better than the experience on the current site https://flathub.org/apps/search/zero%20results
